api/src/main/resources/application-prod.yml

```diff
@@ -0,0 +1,6 @@
+server:
+  port: 8080
+
+integration:
+  port: 9090
```

Comment:
wywalić integration.port. Jeżelibyśmy potrzebowali to wrzucimy taki sam pliczek w integration/src/main/resources, ale tam narazie nie mamy żadnych potrzebnych propertasów